rust_src/src/globals.rs

```diff
@@ -1,5 +1,5 @@
-#![allow(dead_code)] // https://github.com/rust-lang/rust/issues/38780
```

Comment:
I think this comment is worth keeping -- we only have this at the module level due to the bug linked. We can remove this in 1.16.